lib/game_team.rb

```diff
@@ -116,6 +120,31 @@ def self.worst_offense
     total_goals_per_team= Hash[grouped_team.keys.zip(team_averaged_goals)]
     total_goals_per_team.key(total_goals_per_team.values.min)
   end
+
+  def self.most_goals_scored(team_id)
+  total_game_teams_per_team_id = GameTeam.find_by_team(team_id)
```

Comment:
Is calling `GameTeam` here the same as calling `all`?

Reply:
yes